### PR TITLE
add commands to run Rapid

### PR DESCRIPTION
**Pull request by elharo**
Eventually we'll merge this with prepare_release.sh

This is an intermediate step to figure out if this works.

### CODE REVIEW

File: scripts/release_bom.sh [hunk 1]
```diff
@@ -0,0 +1,114 @@
+#!/bin/bash -
+# Usage: ./prepare_release.sh <dependencies|bom> <release version>
```
Comment (suztomo) [inline scripts/release_bom.sh:2]: script name

File: scripts/release_bom.sh [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  echo "$(tput setaf 2; tput bold)$1$(tput sgr0)"
+}
+
+Die() {
+  EchoRed "$1"
+  exit 1
+}
+
+DieUsage() {
+  Die "Usage: ./prepare_release.sh <dependencies|bom> <release version> [<post-release-version>]"
```
Comment (suztomo) [inline scripts/release_bom.sh:19]: Script name

**Review by suztomo — APPROVED** [review]

File: scripts/release_bom.sh (hunk 2 above)
Comment (elharo) [reply scripts/release_bom.sh:19]: done

**Review by elharo — COMMENTED** [review]
ERROR: 'release-10.9.12-bom' is not a valid CitC alias, and cannot be used with 'g4d -f'.

File: scripts/release_bom.sh (hunk 1 above)
Comment (elharo) [reply scripts/release_bom.sh]: done

File: scripts/release_bom.sh [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+git push origin v${VERSION}-${SUFFIX}
+git push --set-upstream origin ${VERSION}-${SUFFIX}
+
+# Create the PR
+gh pr create -t "Release ${VERSION}-${SUFFIX}" -b "Release ${VERSION}-${SUFFIX}"
+
+# File a PR on Github for the new branch. Have someone LGTM it, which gives you permission to continue.
+EchoGreen 'Ask someone to approve this PR.'
+
+p4 g4d -f release-${VERSION}-${SUFFIX}
```
Comment (elharo) [inline scripts/release_bom.sh:108]: period turns out not to be an allowed character in CITC client names, but we can substitute an underscore.

**Comment by elharo** [comment]
java.lang.AssertionError: 
Baseline BOM: com.google.cloud:libraries-bom:12.1.0
Newly introduced problems:
java.nio.ByteBuffer's method position(int) is expected to return java.nio.ByteBuffer but instead returns java.nio.Buffer;
  referenced by 8 class files
    com.google.protobuf.AllocatedBuffer (com.google.protobuf:protobuf-java:3.13.0)
    com.google.protobuf.BinaryWriter (com.google.protobuf:protobuf-java:3.13.0)
    com.google.protobuf.ByteBufferWriter (com.google.protobuf:protobuf-java:3.13.0)
    com.google.protobuf.CodedInputStream (com.google.protobuf:protobuf-java:3.13.0)
    com.google.protobuf.CodedOutputStream (com.google.protobuf:protobuf-java:3.13.0)
    com.google.protobuf.IterableByteBufferInputStream (com.google.protobuf:protobuf-java:3.13.0)
    com.google.protobuf.NioByteString (com.google.protobuf:protobuf-java:3.13.0)
    com.google.protobuf.Utf8 (com.google.protobuf:protobuf-java:3.13.0)
java.nio.ByteBuffer's method limit(int) is expected to return java.nio.ByteBuffer but instead returns java.nio.Buffer;

**Comment by suztomo** [comment]
CircleCI occasionally fails with the error somehow. Retry usually works.

I just filed https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/1718 but not sure how to debug CircleCI environment.

**Pull request merged**